.pre-commit-config.yaml

```diff
@@ -1,18 +1,8 @@
 repos:
-- repo: local
-  hooks:
-  - id: tox
-    name: tox
-    stages: [push]
-    language: system
-    entry: tox
-    types: [python]
-    pass_filenames: false
-
-  - id: tox_integ_tests
-    name: tox integ tests
-    stages: [push]
-    language: system
-    entry: tox -e integ-tests
-    types: [python]
-    pass_filenames: false
```

Comment:
Why remove these? We want the tests to pass before committing.

Reply:
Removing integ tests makes sense, but I think unit tests are still good to have.